src/orthography.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Orthography switching.
+ */
+
+const AVAILABLE_ORTHOGRAPHIES = new Set(['Cans', 'Latn', 'LatnXMacron'])
+
+/**
+ * Listen to ALL clicks on the page, and change orthography for elements that
+ * have the data-orth-switch.
+ */
+export function registerEventListener() {
+  // Try to handle a click on ANYTHING.
+  // This way, if new elements appear on the page dynamically, we never
+  // need to register new event listeners: this one will work for all of them.
+  document.body.addEventListener('click', handleClickSwitchOrthography)
+}
+
+/**
+ * Changes the orthography of EVERYTHING on the page.
+ */
+export function changeOrth (which) {
+  if (!AVAILABLE_ORTHOGRAPHIES.has(which)) {
+    throw new Error(`tried to switch to unknown orthography: ${which}`)
+  }
+
+  let elements = document.querySelectorAll('[data-orth]')
+  let attr = `orth${which}`
+  for (let el of elements) {
+    let newText = el.dataset[attr]
+    if (newText) {
+      el.innerText = newText
+    }
+  }
+}
+
+/**
+ * Switches orthography and updates the UI.
+ */
+function handleClickSwitchOrthography(evt) {
+  let target = evt.target
+  // Determine that this is a orthography changing button.
+  if (target.dataset.orthSwitch === undefined) {
+    return
+  }
+
+  // target is a <button data-orth-swith value="ORTHOGRAPHY">
+  let orth = target.value
+  changeOrth(orth)
+
+  updateUI(target)
+}
+
+/**
+ * Updates the UI following an orthography switch.
+ * Assumes the following HTML:
+ *
+ *  <details>
+ *    <summary>CURRENT ORTHOGRAPHY</summary>
+ *    <ul>
+ *      <li class="menu-choice menu-choice--selected">
+ *        <button data-orth-switch value="ORTH">CURRENT ORTHOGRAPHY</button>
+ *      </li>
+ *      <li class="menu-choice">
+ *        <button data-orth-switch value="ORTH">DIFFERENT ORTHOGRAPHY</button>
+ *      </li>
+ *    </ul>
+ *  </details>
+ */
```

Comment:
Regarding the commented block from lines 53 - 28: are we keeping it for later usage or no? If no, perhaps we remove it?

Reply:
This commented code is not intended to be used! It's supposed to be documentation in order to follow what the JavaScript in `updateUI()` believes the HTML to look like. In general, I try to avoid committing commented out code.

Is there anyway I can make it clear that this code is documentation and, you know, actual real code?

Reply:
Ooh I see, my mistake! In that case, totally good – I'll blame it on me being rusty after a few days away 😅 s